docs/source/user_guide/faq.rst

```diff
@@ -0,0 +1,118 @@
+fastplotlib FAQ
+===============
+
+What is `fastplotlib`?
+----------------------
+
+    `fastplotlib` is a scientific plotting library built on top of the `pygfx <https://github.com/pygfx/pygfx>`_ rendering engine
+    that leverages new graphics APIs and modern GPU hardware to create fast and interactive visualizations.
+
+
+What can I do with `fastplotlib`?
+---------------------------------
+
+    `fastplotlib` allows for:
+        - GPU accelerated visualization
+        - interactive visualization via an intuitive and expressive API
+        - rapid prototyping and algorithm design
+        - easy exploration and fast rendering of large-scale data
+        - design, develop, evaluate and ship machine learning models
+
+Do I need a GPU?
+----------------
+
+    Technically no, you can perform limited software rendering on linux using lavapipe.
+    However, `fastplotlib` is intentionally built for realtime rendering using the latest GPU technologies, so we
+    strongly recommend that you use a GPU. With that being said, an integrated GPU should be sufficient for many
+    use-cases.
```

Comment:
maybe start with "integrated GPUs such as those found in most modern laptops is sufficient for many use cases..."